### PR TITLE
fix: regex is failing on a proper verification code

### PR DESCRIPTION
**Pull request by hphoeksma**
Fixes: #180

### CODE REVIEW

**Review by jonnitto — APPROVED** [review]
Fine by reading

**Comment by Benjamin-K** [comment]
Google now also has an underscore sometimes. Maybe also dashes? So i think we should update the regex to match such verification codes to. Sth. like:

``` regularExpression: '/^[a-zA-Z0-9_-]+$/'```

File: Configuration/NodeTypes.Mixin.GoogleSiteVerification.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@
           position: 100
       validation:
         'Neos.Neos/Validation/RegularExpressionValidator':
-          regularExpression: '/^[a-z0-9]$/i'
+          regularExpression: '/^[a-zA-Z0-9]+$/'
```
Comment (jonnitto) [inline Configuration/NodeTypes.Mixin.GoogleSiteVerification.yaml]:
```suggestion
          regularExpression: '/^[a-zA-Z0-9_-]+$/'
```

**Review by jonnitto — CHANGES_REQUESTED** [review]

**Review by jonnitto — APPROVED** [review]

**Comment by jonnitto** [comment]
https://github.com/neos/neos-seo/releases/tag/3.3.5

**Pull request merged**